kafka/init.lua

```diff
@@ -274,6 +274,22 @@ function Producer:produce_async(msg)
     return err
 end
 
+function Producer:init_transactions(timeout)
```

Comment:
I think we should do separate TransactionalProducer at least on lua wrapper level.